### DISCUSSION TITLE
[Announcement] KCLOUD VM Allocation

### DISCUSSION THREAD

**Discussion by goodtaeeun** [prosyslab-classroom/cs424-program-reasoning#298] [Announcements]
Hi all,

Please check the KLMS announcement for the KCLOUD VM allocation.
Refer to the [guide](https://docs.google.com/presentation/d/1rS39M1dui3khqTc6joWniNrxH3x87nGgOrWHiMAL4DU/edit?usp=sharing) for how to use the VM.

**Comment by RobertoCornacchiari** [comment]
Hi,
it is not too clear to me if we have to use the VM or if it is just an alternative to our personal PCs. If we have to, shall we install there all our environment (like VScode). Sorry for the question, I'm not really used to using virtual machines.

**Comment by goodtaeeun** [comment]
Hi, as you can find in the guide, we recommend two settings for the homework, VM or the Docker image that we provide. Setting your own environment on your PC is less recommended because we can not help you with the troubleshooting. FYI, the VM is simply a remote server, so you can access them via VS Code.

**Comment by uk-jeong** [comment]
@goodtaeeun Hi! My VM ( x.168 ) has been reset. Is there any issue for this?
( Is it necessary for me to contact the KCLOUD staff about this matter? )

**Comment by goodtaeeun** [comment]
Hi @uk-jeong, can you elaborate on the problem you refer to by 'reset'?

**Comment by uk-jeong** [comment]
@goodtaeeun 안녕하세요. 할당해주신 VM 에서 HW1을 완료했었고 조금전 다시 코드를 보기위해 접속했더니 비밀번호와 내부 파일들이 첫 접속과 동일한 상태로 초기화 되어있었습니다.

VM 에서 과제를 제외한 행동은 하지않았어서 복구에 큰 문제가 없지만 혹시 초기화된 원인을 알 수 있을까요?

**Comment by goodtaeeun** [comment]
Oh, I am sorry to hear that. I have heard some other cases where the password is reset to the initial password. However, the fact that files are also reset is new to me. I am sorry but I suggest you contact the KCLOUD management for this issue. Try the following email account (kcloud@kaist.ac.kr)

**Comment by uk-jeong** [comment]
@goodtaeeun I will contact the KCLOUD management for this issue. Thank you for the confirmation!
